content/docs/practices/remote_write.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Remote Write Tuning
+sort_rank: 8
+---
+
+# Remote write tuning
+
+Prometheus implements sane defaults for remote write, but many users have
+different requirements and would like to optimize their remote settings.
+
+This page describes the tuning parameters available via the [remote write
+configuration.](/docs/prometheus/latest/configuration/configuration/#remote_write)
+
+## Remote write characteristics
+
+Each remote write destination starts a queue which reads from the write-ahead
+log (WAL), writes the samples into a queue owned by a shard, which then sends a
```

Comment:
Might be good to clarify that these queues are in-memory only ("writes the samples into a queue" sounds like it's on-disk queues). At least that's how I assume it still works.